.eslintrc.js

```diff
@@ -1,6 +1,7 @@
 module.exports = {
     extends: ["hive/vue", "hive/prettier"],
     rules: {
-        "vue/no-v-html": ["off"]
+        "vue/no-v-html": ["off"],
+        "linebreak-style": "off"

```

Comment:
Let's not change this config, it standardizes our line breaks.

Reply:
for some reason I was always getting this error when this rule was enabled by default:

`expected linebreaks to be 'CRLF' but found 'LF'.` and this has something to do with the OS's (afaik), which can be different and bring this error more often

Reply:
Yes, Linux and MacOS use LF line endings and Windows uses CRLF.

Simple run "npm run pretty" before commiting.